From 5eeab344ea1671fab6b70d5914a8cb4e64f3d45b Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Sun, 22 Apr 2018 03:10:36 +0200
Subject: [PATCH 10/12] lineage: extract_utils: Fix rootfs targets after
 a48b9fe9b6c25746940a2410db640d5e5438363d

* By stripping away the target, rootfs targets get classified
  as normal bin files, so the resulting target entry is wrong

Change-Id: I0ab921d66879675aa4c6582d785e8868d6724897
---
 build/tools/extract_utils.sh | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 007895ad..4e65d586 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -149,7 +149,11 @@ function target_args() {
 function prefix_match() {
     local PREFIX="$1"
     for LINE in "${PRODUCT_PACKAGES_LIST[@]}"; do
-        local FILE=$(target_file "$LINE")
+        if ! [[ "$LINE" =~ "rootfs" ]]; then
+            local FILE=$(target_file "$LINE")
+        else
+            local FILE="$LINE"
+        fi
         if [[ "$FILE" =~ ^"$PREFIX" ]]; then
             printf '%s\n' "${FILE#$PREFIX}"
         fi
-- 
2.17.0

